frame/staking-rewards/src/lib.rs

```diff
@@ -865,7 +958,16 @@ pub mod pallet {
 			}
 		}
 
+		// Review [Andy]
+		// this is probably best being batched which I guess is what ReleaseRewardsPoolsBatchSize
+		// is intended for, spelling here `accumulate`
```

Comment:
spelling has been fixed since you forked, block level fold is low priority for now as we will only have a few reward pools in practice